src/SettingsView/SettingsView.js

```diff
@@ -10,18 +10,32 @@ import AppStyles from "./../AppStyles";
 import {Card, CardHeader, CardText, CardActions} from 'material-ui/Card';
 import IconButton from 'material-ui/IconButton';
 import IconAdd from 'material-ui/svg-icons/content/add';
-import ErrorMessage from './../Utils/ErrorMessage.js';
 import config from './../config/config';
+import Notification from './../Utils/Notification';
 
 class SettingsView extends ConnectionComponent {
 
 	constructor() {
 		super();
-		this.state = {
-			searchText: ''
-		};
+		if(!window.sessionStorage.getItem('notificationMessage')) {

```

Comment:
Dublicated code. Determine snackbarMessage & snackbarMode first and set the state afterwards.